keyboards/system76/launch_2/launch_2.c

```diff
@@ -286,3 +288,42 @@ void bootloader_jump(void) {
     // finally, jump to bootloader
     asm volatile("jmp 0xFC00");
 }
+
+void application_check_jump(void) {
+
+    // Disable all peripherals on AT90USB646
+    UDCON = 1;
+    USBCON = (1<<FRZCLK);  // disable USB
+    UCSR1B = 0;
+    _delay_ms(5);
+
+    EIMSK  = 0;
+    PCICR  = 0;
+    SPCR   = 0;
+    ACSR   = 0;
+    EECR   = 0;
+    ADCSRA = 0;
+    TIMSK0 = 0;
+    TIMSK1 = 0;
+    TIMSK2 = 0;
+    TIMSK3 = 0;
+    UCSR1B = 0;
+    TWCR   = 0;
+    DDRA   = GPIO_MASK_RESET_USB;
+    DDRB   = 0;
+    DDRC   = 0;
+    DDRD   = 0;
+    DDRE   = 0;
+    DDRF   = 0;
+    PORTA  = GPIO_MASK_RESET_USB;
+    PORTB  = 0;
+    PORTC  = 0;
+    PORTD  = 0;
+    PORTE  = 0;
+    PORTF  = 0;
+
+    usb_mux_init();
+
+    // finally, jump to application_jump_check
+    asm volatile("jmp 0xF0A4");
```

Comment:
How did you get this address?

Reply:
The address is just the location that `application_jump_check` gets compiled to from LUFA's DFU bootloader. I could write a script that scrapes that, so we can find it quickly if it changes in the future for some reason